packages/flutter/lib/src/painting/text_span.dart

```diff
@@ -248,11 +248,6 @@ class TextSpan implements TreeDiagnostics {
     return result;
   }
 
-  @override
```

Comment:
Switched to the standard behavior of a TreeDiagnostics class where toString generates a full single line description and fixed the only usage that called toString passing in an argument.